src/models/resource.js

```diff
@@ -0,0 +1,42 @@
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+  class Resource extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      Resource.belongsTo(models.Objective, { foreignKey: 'objectiveId' });
+    }
+  }
+  Resource.init({
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+    },
+    url: {
```

Comment:
As these will be the frozen (if an AR linked to the Objective is approved) `ObjectiveResource` records, I'm not sure if we want to leave it called `url` or not. Possibly the canonical `Resource` has `canonicalUrl` instead of `url`.

Reply:
That's a really good point about the "canonicalUrl". I'm not at all committed to having separate column names between ObjectiveResource and Resource. Having said that, we don't even have to decide until we have an actual `Resources` table in addition to `ObjectiveResources`.